src/github.rs

```diff
@@ -416,6 +416,18 @@ pub enum ReportedContentClassifiers {
     Spam,
 }
 
+#[derive(Debug, serde::Deserialize, serde::Serialize, Eq, PartialEq)]
+pub enum LockReason {
```

Comment:
Which of this reasons would apply to the use case "tracking issues are just not for discussions"?

Reply:
None of these apply/transmit the idea we want, so I wouldn't use using any of those reason. The reason is optional, I didn't use any of them for this PR.